### PR TITLE
[FIX] Audio: Save music settings to local storage.

### LINKED ISSUE #1086: [BUG] Save sound config
## Summary of Bug

Could you please save our sound config in game?

## Steps to Reproduce

Provide detailed step-by-step instructions for how this bug can be reproduced.

1. TODO

## Expected Behavior

save our sound config in game

## Actual Behavior

Have to mute music everytime i start game.

## Screenshots

**Device Information (please complete the following information):**

- Device: [e.g. iPhone6, Pixel 6a]
- OS: [e.g. iOS, Linux, Windows, Android]
- Browser/App [e.g. firefox, safari, chrome, metamask mobile]

### PR DESCRIPTION
**Pull request by AzaroQL**
# Description
Issue:
Each time user opens/reloads game in non chromium browser music starts playing.
Solution:
I have added musicPaused and musicVolume to local storage, next to audioMuted setting.

Fixes #1086

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual test in firefox and chrome browsers.
yarn test.

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [n/a] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [n/a] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/ui/AudioPlayer.tsx [hunk 1]
```diff
@@ -17,43 +17,52 @@ import {
...
+      0.1
+    ),
+    step: 0.1,
+    max: 1,
+    min: 0,
+  });
   const [visible, setIsVisible] = useState<boolean>(false);
-  const [muted, setMuted] = useState<boolean>(getCachedAudioMuted());
-  const [isPlaying, setPlaying] = useState<boolean>(true);
+  const [audioMmuted, setAudioMuted] = useState<boolean>(
```
Comment (Rofrtd) [inline src/components/ui/AudioPlayer.tsx:57]: Typo here on `audioMmuted`.

File: src/components/ui/AudioPlayer.tsx [hunk 2]
```diff
@@ -17,43 +17,52 @@ import {
...
+    getCachedAudioSetting<boolean>(AudioLocalStorageKeys.audioMuted, false)
+  );
+  const [musicPaused, setMusicPaused] = useState<boolean>(
+    getCachedAudioSetting<boolean>(AudioLocalStorageKeys.musicPaused, true)
```
Comment (Rofrtd) [inline src/components/ui/AudioPlayer.tsx:61]: Could we make it playing by default instead of paused?

**Review by Rofrtd — CHANGES_REQUESTED** [review]
Thanks for your contribution.
Nice fixes, just left a feel comments:

File: src/components/ui/AudioPlayer.tsx (hunk 1 above)
Comment (AzaroQL) [reply src/components/ui/AudioPlayer.tsx]: Fixed.

File: src/components/ui/AudioPlayer.tsx (hunk 2 above)
Comment (AzaroQL) [reply src/components/ui/AudioPlayer.tsx]: Oh, good catch, intention was to autoplay music on non chromium browsers, but It looks like I forgot that value when refactoring from isPlaying to musicPaused. Fixed. :)

**Review by Rofrtd — APPROVED** [review]
Nice one mate! 
LGTM. :shipit:

**Pull request merged**